crates/noirc_frontend/src/parser/labels.rs

```diff
@@ -0,0 +1,40 @@
+use std::fmt;
+
+use crate::token::TokenKind;
+
+#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord)]
```

Comment:
Can you put a comment for what this is? Something like "constants used in the parser"

Not sure about the name `ParserLabel` but want to avoid bikeshedding :)